renderdoc/driver/vulkan/vk_debug.cpp

```diff
@@ -1423,6 +1423,13 @@ uint32_t VulkanReplay::PickVertex(uint32_t eventId, int32_t w, int32_t h, const
   return ret;
 }
 
+VulkanCreationInfo::Image VulkanDebugManager::GetImageInfo(ResourceId img)
```

Comment:
This return value would be better returning a const reference I think, rather than a copy.

Reply:
Done